### ISSUE TITLE
Problems with building the apk

### ISSUE THREAD

**Issue by 0iras0r** [dyne/ZShaolin#2]
Hi ! I have problems to set enviromental variable ZHOME, can you explain the steps in details?
I receive 'module not found'
I need to build this application, because I don't use Playstore.
Thanks in advance.

**Comment by jaromil** [comment]
Docs on building are in the making, but there are more priorities pending.

**Comment by dragroot** [comment]
When i do -
**$ ./zmake build**
[!] Set ZHOME environment to Zshaolin path

**$ export ZHOME=/home/anon/zshaolin**

**$ ./zmake build**
ZHOME=/home/anon/zshaolin
.  Enter /home/anon/zshaolin/build
[!] Module not found: all

**$ export ZHOME=/home/anon/zshaolin/build**

**$ ./zmake build**
ZHOME=/home/anon/zshaolin/build
.  Enter /home/anon/zshaolin/build
[!] Module not found: all

Please explain it...

**Comment by jaromil** [comment]
`./zmake build name-of-module` (a module is a dir inside the modules dir)

Beware this is mostly left as-is and unmaintained. The apks I've released in the past still work.
So many things have changed in Android since version 2 that I can hardly catch up, but if I'll ever find the time I'll make a new build that is completely free. Meanwhile I also recommend having a look at Termux (also available in the f-droid market I believe).

**Comment by dragroot** [comment]
Thanks for your information. 
And about turmux, it's not available for Android 4.4.4 (Kitkat).

**Comment by jaromil** [comment]
Yes, ZShaolin is old-school and somehow less resource intensive. Well if you feel like you are getting somewhere and determined to revive this project I'm available to assist you more. You'll find me also on irc.dyne.org channel `#dyne`.
